docs/DeveloperGuide.adoc

```diff
@@ -800,18 +801,63 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |add a new tutor/student |
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |user |delete a tutor/student |remove entries that I no longer need
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |user |find a tutor/student by name |locate details of persons without having to go through the entire list
```

Comment:
you may want to specify the "type of user" for whom the user story is relevant rather than using "user" alone